src/modules/Developers/Developers.tsx

```diff
@@ -0,0 +1,104 @@
+import {
+  Heading,
+  Text,
+  UnorderedList,
+  ListItem,
+  Image,
+  Box,
+  Card,
+  CardBody,
+  CardFooter,
+  Flex,
+  Container,
+} from '@chakra-ui/react';
+
+const devTeamInfo = [
+  {
+    id: 0,
+    fullName: 'Alejandro Coronado',
+    img: 'https://www.nefrologiaenlinea.com/static/media/jacc.0ca55152d92bcd4c0d97.jpeg',
+    role: [
+      'Product Development',
+      'Lead Developer',
+      'Web Architect',
+      'Frontend Development',
+      'Backend Development',
+    ],
+    linkedinUrl: 'https://www.linkedin.com/in/jorge-alejandro-coronado-cuesta/',
+  },
+  {
+    id: 1,
+    fullName: 'Hicham-Zaadla',
+    img: 'https://avatars.githubusercontent.com/u/99765449?v=4',
+    role: ['Frontend Development', 'Design'],
+    linkedinUrl: '',
```

Comment:
Hello @AlbertLeipzig , here is my linkedin `https://www.linkedin.com/in/hicham-zaadla/`  :)